src/query/service/src/servers/http/http_services.rs

```diff
@@ -97,7 +98,8 @@ impl HttpHandler {
     async fn build_router(&self, sock: SocketAddr) -> impl Endpoint {
         let ep_v1 = Route::new()
             .nest("/query", query_route())
-            .at("/login", post(login_handler))
+            .at("/session/login", post(login_handler))
+            .at("/session/renew", post(renew_handler))
```

Comment:
We can consider add a `logout` handler, so that client can delete current token.